pims/utils/sort.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
No reason to spilt them (but they don't all fit on one line and comply with
pep8)

On Mon, Jan 12, 2015, 07:36 Casper van der Wel <notifications@github.com>
wrote:

> In pims/utils/sort.py
> <https://github.com/soft-matter/pims/pull/127#discussion-diff-22787272>:
>
> > @@ -0,0 +1,31 @@
> > +# -*- coding: utf-8 -*-
> > +
> > +from __future__ import unicode_literals
> > +from __future__ import division
> > +from __future__ import absolute_import
> > +from __future__ import print_function
>
> I prefer the four on one line. Or are there reasons to split them?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/soft-matter/pims/pull/127/files#r22787272>.
>